lib/modules/neverEndingReddit.js

```diff
@@ -271,14 +262,15 @@ modules['neverEndingReddit'] = {
 		if (this.pauseAfterPages === null) {
 			this.pauseAfterPages = parseInt(modules['neverEndingReddit'].options.pauseAfterEvery.value, 10);
 		}
-
 		if ((this.pauseAfterPages > 0) && (currPageNum % this.pauseAfterPages === 0)) {
-			this.togglePause(true);
-			var notification = [];
-			notification.push('Time for a break!');
-			notification.push('Never-Ending Reddit was paused automatically. ' + modules['settingsNavigation'].makeUrlHashLink('neverEndingReddit', 'pauseAfterEvery', '', 'gearIcon'));
-			notification = notification.join('<br><br>');
-			setTimeout(modules['notifications'].showNotification.bind(RESUtils, notification, 5000));
+			this.togglePause('pauseAfterEvery');
+			var message = '<p>Time for a break!</p>';
+			message += '<p>Never-Ending Reddit has been paused because you\'ve passed ' + this.pauseAfterPages + ' pages. You can change this number in RES settings.</p>';
+			modules['notifications'].showNotification({
+				moduleID: modules['neverEndingReddit'].moduleID,
```

Comment:
You can add in `optionKey: 'pauseAfterEvery'` to make the gear link to that option.